policy/approval/approve_test.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type Requires = common.Requires
```

Comment:
I am removing this because we are no longer using the same `Requires` struct for approval and disapproval. So there is no need to put `Requires` in common package.